bin/install.sh

```diff
@@ -130,7 +130,10 @@ function installResources() {
     [ -d "${path}" ] || continue # skip if not a directory
     local directory
     directory="$(basename "${path}")"
-    resources+=("${directory}")
+    # Check if directory contains Chart.yaml File:
+    if [[ $(find "$resource_directory"/"$directory"/Chart.yaml 2>/dev/null | wc -l) -gt 0 ]]; then
```

Comment:
Why is `if [[ -f "$resource_directory/$directory/Chart.yaml" ]]; then` not sufficient here?